connectomics/discfibers/ea_discfibers_merge_pathways.m

```diff
@@ -28,6 +29,10 @@
   pathway_list{k} = regexprep(pathway_list{k}, '_', ' ');
   
   fiber_file = load(fullFileName);
+  % skip the adjacency matrix file
+  if contains(fullFileName, '_ADJ')
+      continue 
+  end
```

Comment:
this is not necessary since `myFiles` was already filtered in line 10?

Reply:
no, not necessary anymore

Reply:
No, not necessary anymore